CHANGELOG.md

```diff
@@ -1,3 +1,12 @@
+## [1.6.4](https://github.com/algolia/autocomplete/compare/v1.6.3...v1.6.4) (2022-06-21)
+
+
+### Bug Fixes
+
+* **autocomplete-js:** leave the modal open on reset on pointer devices ([#987](https://github.com/algolia/autocomplete/issues/987)) ([3e387e6](https://github.com/algolia/autocomplete/commit/3e387e6e7dea7de46acbaf4e220bbd1e568f4ea2))
```

Comment:
This is such a fundamental change that I think we should bump it to a minor, no?

Reply:
Behavior-wise, it doesn't change anything for `autocomplete-js` users. Core users get a new environment prop (`onMouseDown`) but I don't know if we should treat this as a new "feature".

Reply:
I wouldn't want a batch bump to break my app, and I think this could still happen in some edge cases. It's more about risk than feat/fix here to me.